sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableReadOptions.java

```diff
@@ -0,0 +1,73 @@
+package org.apache.beam.sdk.io.gcp.bigtable;
```

Comment:
Needs an ASF license

Reply:
Done. Thanks for checking the test results btw, I tried to figure out what's wrong from the failing tests but couldn't find any error logs or stacktraces.